docs/markdown/Release-notes-for-0.58.0.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Release 0.58.0
+short-description: Release notes for 0.58.0
+...
+
+# New Features
+
+## `summary()` accepts features
+
+Build feature options can be passed to `summary()` as the value to be printed.
```

Comment:
You should create a new file in that folder for your snippet. Files are concat for the release.

Reply:
I'm unsure what you mean here. What file should I make?

Reply:
Sorry, that's indeed impossible to discover for new contributors... Our process is to create a new file in `docs/markdown/snippets/` all those files there will be concat to make `docs/markdown/Release-notes-for-0.58.0.md` when doing the release.